ex22.py

```diff
@@ -0,0 +1,20 @@
+file = "namesfile.txt"
+try:
+    file = open(file)
+except FileNotFoundError:
+    print(f"No such file found named '{file}'.")
+    exit(1)
+
+line=file.readline()
+
+count = {}
+while line:
+    name = line.strip()
```

Comment:
Do we really need the name over here? Can it not be avoided? Try to avoid unnecessary variable declaration.